sttp/client/src/main/scala/endpoints4s/sttp/package.scala

```diff
@@ -1,4 +0,0 @@
-package endpoints4s
-
-/** Client interpreter using Sttp */
-package object sttp
```

Comment:
Would you mind adding back this file, I think it’s nice to have this documentation when we browse the packages: https://endpoints4s.github.io/api/endpoints4s/sttp/index.html